swig/include/python/gdal_python.i

```diff
@@ -2485,6 +2485,17 @@ mapGRIORAMethodToString = {
     gdalconst.GRIORA_Gauss: 'gauss',
 }
 
+def _addCreationOptions(new_options, creationOptions):
```

Comment:
This is both modifying `new_options` and returning it, which could be unexpected.

Reply:
good point, I was following a bit too much "papszFoo = CSLAddString(papszFoo, pszBar)"
Updated to no longer return a value and just update in place new_options:
```
def _addCreationOptions(new_options, creationOptions):
    """Update new_options with creationOptions formatted as expected by utilities"""
```